src/shared/proto/vision.proto

```diff
@@ -4,14 +4,14 @@ import "shared/proto/geometry.proto";
 
 message RobotStateMsg
 {
-    PointMsg global_position_meters                  = 1;
-    AngleMsg global_orientation_radians              = 2;
-    VectorMsg global_velocity_meters_per_sec         = 3;
-    AngleMsg global_angular_velocity_radians_per_sec = 4;
+    PointMsg global_position                   = 1;
+    AngleMsg global_orientation                = 2;
+    VectorMsg global_velocity                  = 3;
+    AngularVelocityMsg global_angular_velocity = 4;
 }
 
 message BallStateMsg
 {
-    PointMsg global_position_meters          = 1;
-    VectorMsg global_velocity_meters_per_sec = 2;
+    PointMsg global_position  = 1;
+    VectorMsg global_velocity = 2;
```

Comment:
I don't think it's worth changing it since being explicit will help avoid confusion. In any case, I don't think it's necessary for it to be done in this PR